packages/babel-preset/index.js

```diff
@@ -179,8 +179,6 @@ function preset(api, explicitOptions = {}) {
 
       // - convenience plugins --
       require.resolve('babel-plugin-dev-expression'),
-      require.resolve('@babel/plugin-proposal-optional-chaining'),
```

Comment:
these now in shipped proposals?

Reply:
https://babeljs.io/blog/2020/01/11/7.8.0#ecmascript-2020-default-support-10811-https-githubcom-babel-babel-pull-10811-10817-https-githubcom-babel-babel-pull-10817-10819-https-githubcom-babel-babel-pull-10819-10843-https-githubcom-babel-babel-pull-10843

stage 4 now